SPECS/python-jinja2/python-jinja2.spec

```diff
@@ -3,15 +3,15 @@
 %{!?python3_version: %define python3_version %(python3 -c "import sys; sys.stdout.write(sys.version[:3])")}
 
 Name:           python-jinja2
-Version:        2.10.1
+Version:        2.11.3
 Release:        1%{?dist}
 Url:            https://jinja.pocoo.org/
 Summary:        A fast and easy to use template engine written in pure Python
 License:        BSD
 Group:          Development/Languages/Python
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        https://files.pythonhosted.org/packages/93/ea/d884a06f8c7f9b7afbc8138b762e80479fb17aedbbe2b06515a12de9378d/Jinja2-%{version}.tar.gz
+Source0:        https://files.pythonhosted.org/packages/4f/e7/65300e6b32e69768ded990494809106f87da1d436418d5f1367ed3966fd7/Jinja2-%{version}.tar.gz
```

Comment:
For convenience, PyPI also provides the following URL format: `https://files.pythonhosted.org/packages/source/<first letter of package name>/<package name>/<package name>-<package version>.tar.gz`. For this package, it would look like the following:

`https://files.pythonhosted.org/packages/source/J/Jinja2/Jinja2-2.11.3.tar.gz`